File: src/app/models/Activity.ts
import { BaseModel } from "./BaseModel";

export class Activity
    extends BaseModel
{
    public name: string = "";
    public startDate: Date = new Date();
    public endDate: Date = new Date();

    constructor() {
        super();
    }

    public static override toUrlResource(): string {
        return "Activity";
    }
}

File: api-netcore/TasksReportManager/TasksReportManager.EntitiesModel/Activity.cs
namespace TasksReportManager.EntitiesModel
{
  public class Activity
    : EntityBase
  {
    public string Name { get; set; } = string.Empty;
    public DateTime StartDate { get; set; } = DateTime.Now;
    public DateTime? EndDate { get; set; } = null!;

  }
}
